### PR TITLE
Feature/fsar 143 service links

### PR DESCRIPTION
**Pull request by Yinuo-M**
1. Create service links according to the design.
2. Title and description are optional.
3. The admin can specify the number of columns to be shown when the screen is wider than 1440px.
4. Add the element.closest polyfill for IE11 for the collapsible checkboxes component

### CODE REVIEW

File: src/components/landing/ServiceLinks/serviceLinks.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  }
+
+  &__anchor {
+    padding: 1rem 3.5rem 1rem 1rem;
+    margin: 0;
+    color: var(--fsa-primary_black);
+    display: block;
+    height: 100%;
+    width: 100%;
+    background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg width='28px' height='16px' viewBox='0 0 28 16' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Ctitle%3EGroup 5%3C/title%3E%3Cg id='Symbols' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd' stroke-linecap='round'%3E%3Cg id='service-card/Desktop' transform='translate(-253.000000, -25.000000)' stroke='%23007C75' stroke-width='2'%3E%3Cg id='Group-5' transform='translate(254.000000, 26.000000)'%3E%3Cline x1='0' y1='7.25' x2='25' y2='7.25' id='Stroke-1'%3E%3C/line%3E%3Cpolyline id='Stroke-3' points='18 0 25 6.99993305 18 14'%3E%3C/polyline%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
```
Comment (c-ancia) [inline src/components/landing/ServiceLinks/serviceLinks.scss:104]: Can you pull from main and use the image file from now on please? I believe I fixed the image export :)

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,11439 @@
+Arguments:
```
Comment (c-ancia) [inline yarn-error.log:1]: Can you remove this file please?

File: yarn.lock [hunk 3]
```diff
@@ -4,27 +4,25 @@
 
 "@babel/code-frame@7.10.4":
```
Comment (c-ancia) [inline yarn.lock]: Since we have the yarn.lock file, can you remove the package-lock.json file please?

File: webpack.config.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```
Comment (c-ancia) [inline webpack.config.js:1]: I changed a bit this file so you'll have to fix the conflicts when you pull from main :)

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/landing/ServiceLinks/serviceLinks.scss (hunk 1 above)
Comment (Yinuo-M) [reply src/components/landing/ServiceLinks/serviceLinks.scss]: Done

File: yarn-error.log (hunk 2 above)
Comment (Yinuo-M) [reply yarn-error.log]: Removed

File: yarn.lock (hunk 3 above)
Comment (Yinuo-M) [reply yarn.lock:5]: Removed

**Review by c-ancia — APPROVED** [review]

**Pull request merged**